client/status/common.go

```diff
@@ -26,8 +26,15 @@ var (
 	ErrNodeUnderMaintenance NodeUnderMaintenance
 )
 
+// APIErrorDetails gives unified method to get details and determine what happened from certain error.
+type APIErrorDetails interface {
+	Code() status.Code
```

Comment:
we usually describe all public interface's methods

Reply:
You trues, [here](https://github.com/nspcc-dev/neofs-sdk-go/pull/413#discussion_r1197602737) we trying to decide - is it usefull or remove at all